vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -41,27 +41,17 @@ void AccessibilityInfo::setAccessibilityFocus(double /*reactTag*/) noexcept {
 
 void AccessibilityInfo::announceForAccessibility(std::string announcement) noexcept {
   m_context.UIDispatcher().Post([context = m_context, announcement = std::move(announcement)] {
-    xaml::UIElement element{nullptr};
 
-    // Windows requires a specific element to announce from.  Unfortunately the react-native API does not provide a tag
-    // So we need to find something to try to raise the notification event from
+    // Windows requires a specific element to announce from. Unfortunately the react-native API does not provide a tag
+    // So we need to add a temporary control to raise the notification event from.
 
-    if (auto window = xaml::Window::Current()) {
-      element = window.Content();
-    }
-
-    if (!element && react::uwp::Is19H1OrHigher()) {
-      // XamlRoot added in 19H1
-      if (auto xamlRoot = React::XamlUIService::GetXamlRoot(context.Properties().Handle())) {
-        element = xamlRoot.Content();
-      }
-    }
+    auto textBlock = xaml::Controls::TextBlock();
```

Comment:
nit: seems like a UIElement might be enough - https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.frameworkelementautomationpeer.fromelement?view=winrt-19041
Would be a little cheaper to create - can you try that?

Reply:
you can't instantiate UIElement directly, only subclasses

Reply:
Should I leave it as is? Or is there a cheaper subclass I should instantiate instead?

Reply:
I think it might work with a Control

Reply:
Ok looks like Control also can't be instantiated directly only its subclasses.